### DISCUSSION TITLE
How to setup in existing project folder?

### DISCUSSION THREAD

**Discussion by YJDoc2** [skeletonlabs/skeleton#1414] [Q&A] [answered]
Hey, I have a project which I want to re-write completely in svelte+skeleton ui. For that I have deleted everything except few (non-js ; non-npm) files from that dir, and trying to create automatic skeleton install using cli. However, after selecting the options, at the last stage it says cannot install as `Install directory already exists` . I am not trying to add skeleton/svelte to existing project, but want to add it in existing directly because I have a git of that repo in there. Is there any way to do this?

Also if we cannot create project in existing dir, can the error be show even before we select options rather that at very end?

Thanks!

**Comment by Sarenor** [comment, marked as answer]
Sadly I can't move this to skeletonlabs/create-skeleton-app where it probably belongs.
In short: The CLI can't work in a pre-existing directory to make sure it doesn't overwrite things you might have wanted to keep.

If you want to set up Skeleton in a pre-existing projects there's 2 ways:
1) Set up a new project with the CLI and copy over all the things you want to keep
2) Follow the manual installation instructions here: https://github.com/skeletonlabs/skeleton/discussions/1414

**Reply by YJDoc2** [reply]
Hey, apologies, I though this repo would be the correct one for the question. I will mark this as answered, and open a separate one at that repo. Should I open it as an issue instead? 

Thanks a lot for quick answer, I'll do as you have suggested.

Thanks and apologies :)

**Reply by Sarenor** [reply]
All good, we can move this one over, just for future CLI question, that's where the CLI lives :)

No problem and glad to help :)

**Reply by endigo9740** [reply]
@YJDoc2 I'm not sure we'll ever support the CLI for updating an existing project. There's too many complexities for this and potential to break your project. @niktek is the maintainer of the CLI and can speak to this to more detail. You're welcome to create another issue about this on the other repo, but I would not hold my breath about it being implemented anytime soon.

That said, we would love to have this. If we can find a way to reasonably implement it we will. But in the meantime the manual installation instructions are our recommendation for migrating an existing SvelteKit project.
